tests/lib/outputs/csv/output_csv_test.py

```diff
@@ -0,0 +1,204 @@
+from datetime import datetime
+from io import StringIO
+from unittest.mock import Mock
+
+import pytest
+
+from prowler.lib.outputs.common_models import CSV, Finding, Output, Severity, Status
+from prowler.lib.outputs.csv.csv import write_csv
+
+
+@pytest.fixture
```

Comment:
Please create a Test Class here and maybe move it to a folder with the same name as the code within Prowler.

Reply:
Done